src/views/RecIM/components/Forms/ActivityForm/index.jsx

```diff
@@ -273,8 +306,15 @@ const ActivityForm = ({
 
   const handleConfirm = () => {
     setSaving(true);
-    let activityRequest = { ...currentInfo, ...newInfo };
 
+    let imageData = null;
+
+    if (cropperImageData !== null) {
+      imageData = cropperRef.current.cropper.getCroppedCanvas({ width: CROP_DIM }).toDataURL();
+    }
+
+    let activityRequest = { ...currentInfo, ...newInfo };
+    activityRequest.Logo = imageData;
```

Comment:
```suggestion
    let activityRequest = { ...currentInfo, ...newInfo };
    activityRequest.Logo = cropperImageData !== null ? cropperRef.current.cropper.getCroppedCanvas({ width: CROP_DIM }).toDataURL() : null;
```

Personal preference, but someone should second me on this before you commit this one.